interview-questions.md

```diff
@@ -8,48 +8,58 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is Javascript markdown that behaves like a HTML. In simple terms its the meshing of HTML and Javascript in react. 
 
-Researched answer:
+Researched answer: JSX is a syntax extension of Javascript in React. JSX makes it easier to add HTML in React. One difference between HTML and JSX is that JSX needs a return for the output. 
 
 2. What is yarn? What file(s) are modified in a React application when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: Yarn is a package manager for Javascript that contains code being shared by other developers. It is a platform to share code, and install jest. It has thousands of node modules to assist developers. 
 
-Researched answer:
+Researched answer: Yarn, the acronym meaning Yet Another Resource Negiotator. Yarn is an extension system for running big information applications. It is a package manager for Javascript that allows developers and share code in one platform. 
 
 3. What is an anonymous function in JavaScript?
 
-Your answer:
+Your answer: An anonymous function in Javascript is a function that does not have anything declared in the function. 
 
-Researched answer:
+Researched answer: An anonymous function is a function that does not have any name associated with it. Its a function that is declared without any identifier.
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: React state is a variable that holds information, props is an object, it is used to pass the information from one component to another. 
 
-Researched answer:
+Researched answer: React state is a variable that holds information, props (properties) are used to pass arguments as attributes through the component. React doesn’t process data, and does not store algorithms. It takes in data, handles it, and displays it. The use state holds that data/information, and hold it until passed through an argument to be displayed. 
 
 5. What is the DOM?
 
-Your answer:
+Your answer: DOM is Document Object Model, which is the visual representation of code. 
 
-Researched answer:
+Researched answer:  The DOM (Document Object Model) represents the webpage as a structure. Makes it so that you can actively interact with one node instead of refreshing the screen after every user interaction. 
 
 6. STRETCH: Which is the difference between a div and a span?
 
-Your answer:
+Your answer: The div and span tags are not semantic elements since they provide no context as to what is inside of those tags. The difference between the two is that div is used for organization and span is an inline container. 
 
-Researched answer:
+Researched answer: The difference between the div and the span element is that the span tag is used to group elements and content inline, for styling purposes. Div is used for organizing the webpage. 
```

Comment:
Great work in this section!  There are a few answers where you could expand a little more, but overall really good job defining the concepts.